pkg/common/namespace.go

```diff
@@ -0,0 +1,15 @@
+package common
+
+import "fmt"
+
+const namespaceFormat = "%s-%s"
+const namespaceVariable = "namespace"
+const domainVariable = "domain"
+
+// Return kubernetes namespace name
+func GetNamespaceName(mapping, project, domain string) string {
```

Comment:
stylistic nit: instead of using a string for mapping can we use an enum (https://yourbasic.org/golang/iota/) so we have a defined set of mapping modes in one location?

also can we use a switch statement below?